app/export.go

```diff
@@ -175,9 +175,12 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
 		}
 		return false
 	})
+	if err != nil {
+		panic(fmt.Errorf("error when iterating through redelegations: %w", err))
```

Comment:
Maybe instead of using `fmt.Errorf` use `errors.Wrap` from "cosmossdk.io/errors"

Reply:
I'm copying how it's done elsewhere